ContentPatcher/Framework/Tokens/ValueProviders/TranslationValueProvider.cs

```diff
@@ -25,7 +25,7 @@ internal class TranslationValueProvider : BaseValueProvider
         /// <summary>Construct an instance.</summary>
         /// <param name="translationHelper">Gets translations from the content pack's translation folder.</param>
         public TranslationValueProvider(ITranslationHelper translationHelper)
-            : base(ConditionType.I18n, mayReturnMultipleValuesForRoot: false)
+            : base(ConditionType.I18n, mayReturnMultipleValuesForRoot: false, isDeterministicForInput: true) // This is deterministic 99% of the time, but if language is changed, this would need to be invalidated
```

Comment:
Unfortunately this will introduce a bug where content packs don't update correctly when the player changes language on the title screen.
```suggestion
            : base(ConditionType.I18n, mayReturnMultipleValuesForRoot: false)
```